1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_1_Fizzbuzz/src/fizzbuzz.ts

```diff
@@ -1,5 +1,8 @@
 const fizzbuzz = (numbers: number[]) => {
-  return '';
+  return numbers.reduce((acc, num) => {
+    if (num % 3 === 0) return acc + 'fizz';
+    return acc;
+  }, '');
```

Comment:
It looks like to went ahead and took the obvious implementation path here. it's fine for simple problems when you're sure of the solution, but it's better to take smaller steps and use the "Fake it" strategy... until you see 3 repetitions, then you can refactor your code using the obvious implementation strategy. 

I'm not sure the problem statement of FizzBuzz requires the use of an array as an input to the function.